src/Microsoft.OpenApi/Models/OpenApiConstants.cs

```diff
@@ -595,6 +595,16 @@ public static class OpenApiConstants
         /// </summary>
         public static readonly Uri defaultUrl = new Uri("http://localhost/");
 
+        /// <summary>
+        /// Field: V3 JsonSchema Reference Uri
+        /// </summary>
+        public const string v3ReferenceUri = "https://everything.json/components/schemas/";
```

Comment:
I feel like we should have a more appropriate default base URL for here rather than something that is based on the library we are taking a dependency on.  Not a big deal, but something to think about.